bokeh/io/webdriver.py

```diff
@@ -63,6 +63,7 @@ def create_chromium_webdriver() -> WebDriver:
     options = webdriver.chrome.options.Options()
     options.add_argument("--headless")
     options.add_argument("--hide-scrollbars")
+    options.add_argument("--force-device-scale-factor=1")
```

Comment:
I'm not sure what's the best approach, so I implemented two. One is to force DPR to 1 and the other to scale the image to the desired width/height, so that `Plot(width=w, height=h)` generates a PNG of w x h dimensions on all platforms. I would like to establish which approach gives better image quality if the native DPR != 1. To test this, one needs to generate an image with `export_png()` with this line enabled and then another with this line commented out. You should get two PNGs of the same dimensions in both cases (use e.g. `file <filename>.png` to confirm). This is primarily directed towards @bryevdv, but @philippjfr, if you have a system with native DPR != 1, then please try this as well. The alternative is to produce PNGs in "full resolution", which would result in `w*dpr` x `h*dpr` dimensions. This is what save tool does. Any thoughts are welcome.

Reply:
My guess is that setting DPR to 1 and letting the browser do what it does will be best, but I will check out the PR later today.

Reply:
@mattpap  I tested a handful of examples and the versions with DPR set to 1 look clearly better to me.

Reply:
In that case the PR is ready. I will leave both, so that browsers that don't support setting device metrics, can still produce decent results.

Reply:
Firefox supports this as well, but I need to figure out how to set this up from command line.

Reply:
Does that mean you want to wait to merge this?

Reply:
I will submit another PR with this. In fact I already figured this out for another bit of work, but can't find it now.